data/themes/default/style.css

```diff
@@ -146,7 +146,7 @@ PianoRoll {
 	qproperty-noteBorders: false; /* boolean property, set false to have borderless notes */
 	qproperty-selectedNoteColor: #006b65;
 	qproperty-barColor: #078f3a;
-	qproperty-markedSemitoneColor: #06170E;
+	qproperty-markedSemitoneColor: rgba(255, 255, 255, 20);
```

Comment:
Something like this for classic theme too?

Reply:
I always forget about that, will fix